nextflow_schema.json

```diff
@@ -205,14 +205,6 @@
                     "fa_icon": "fas fa-remove-format",
                     "hidden": true
                 },
-                "max_multiqc_email_size": {
```

Comment:
If you remove this part, why not the rest of the multiqc parameters ?

Reply:
on it, thanks for noticing it all